src/server/api/routers/files.ts

```
@@ -282,336 +208,240 @@ async function removeInternalQuestionFile({
 
-      const tFQuestions = tFQ.value.data!;
+        case "true_false":
+          const tFQuestions = await _(
+            // könnte auch ohne _ aber ist besser zum reingewöhnnen
+            databaseInteraction(
+              tx
+                .select()
+                .from(questionTrueFalse)
+                .where(eq(questionTrueFalse.questionId, questionId)),
+            ),
+          );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)